webots_ros2_epuck/protos/E-puck_enu.proto

```diff
@@ -89,7 +89,7 @@ Robot {
                       roughness 0.4
                       baseColorMap ImageTexture {
                         url [
-                          "textures/gctronic_logo.png"
+                          "https://raw.githubusercontent.com/cyberbotics/webots/R2021a/projects/robots/gctronic/e-puck/protos/textures/gctronic_logo.png"
```

Comment:
Shouldn't we target a tag or a commit? This way, if we move/rename textures we will break a lot of things

Reply:
We used the `R2021a` tag in the webots_ros package. That seems to be a good compromise